R/antigen.garnish_predict.R

```diff
@@ -185,9 +186,9 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
 
     dto <- data.table::rbindlist(list(dti, vdt), fill = TRUE, use.names = TRUE)
 
-  # sanity check to make sure no '-' slipped through from blastp
+  # sanity check to make sure no special symbols slipped through from blastp
 
-    dto <- dto[!nmer %like% "-"]
+    dto <- dto[!nmer %like% "-|\\*"]
```

Comment:
I suggest we white list known good here as well, in a single step

Reply:
Good call. Can use Biostrings::AMINO_ACID_CODE[1:20]

Reply:
yea or just hard code it

Reply:
Done